assembl/static2/js/app/utils/timeline.js

```diff
@@ -145,37 +145,33 @@ export const isCurrentPhase = (_phase: Phase) => {
   return currentPhase;
 };
 
-export const getPhaseName = (_timeline: Timeline, _id: string) => {
-  const timeline = _timeline;
-  const phaseId = _id;
-  let phaseName = '';
-  if (!phaseId || !timeline) return phaseName;
-  timeline.forEach((phase) => {
-    if (phase.id === phaseId) {
-      phaseName = phase.title;
-    }
-  });
-  return phaseName;
+function findPhase(timeline: Timeline, phaseId: string): Phase {
```

Comment:
Just curious: in what case do we favour the function keyword over the arrow syntax?

Reply:
I favor the function keyword for functions that are on top level of a file. I think arrow functions are better for callbacks and functions defined at a deeper level (in a render for example). But it's a personal preference I guess.